documentation/crl_refresh.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: "Puppet Server: Automatic CRL refresh"
+canonical: "/puppetserver/latest/crl_refresh.html"
+---
+
+Starting in version 2.8.0, Puppet Server can automatically reload an updated CRL into the running SSL context, so that the revocation of an agent's certificate no longer requires a restart of the service to take effect. Prior versions required an explicit restart or reload of this service to reload the CRL, resulting in some small amount of downtime to effect the revocation of a certificate. With automatic CRL refresh enabled, revocation is now transparent and requires no service downtime.
+
+### Enabling
+
+To enable automatic CRL refresh, modify your Puppet Server services bootstrap configuration file to include the following line:
```

Comment:
Can we specify up front that this should be `/etc/puppetlabs/puppetserver/services.d/ca.cfg`, at least by default?